docker-compose-tests.yml

```diff
@@ -0,0 +1,25 @@
+version: '3.8'
+
+services:
+  app-tests:
```

Comment:
How you use it in tests?

Reply:
No. 
But it should be in order to start tests from it. 
I mean I could update Dockerfile like this : 
`RUN ./gradlew test `
And run tests in docker

Reply:
Ok, got it, bad idea.